kubesae/providers/aws.py

```diff
@@ -54,6 +54,7 @@ def sync_media_tree(
     media_bucket="MEDIA_STORAGE_BUCKET_NAME",
     acl="private",
     local_target="./media",
+    append_path="",
```

Comment:
My only thought would be to make this variable more related to the bucket, like `bucket-path`, so it's clear which path it's referring to within the task.

Reply:
:+1: